src/_update.sh

```diff
@@ -107,8 +107,9 @@ function selfupdate_doUpgrade() {
   debug "Tempdir: $tempDir"
   target="${PLEXTRAC_HOME}/.local/bin/plextrac"
 
-  debug "`curl -w %{url_effective} -L -o $tempDir/$(jq '.name, " ", .browser_download_url' -r <<<$scriptAsset) 2>&1 || error "Release download failed"`"
-  debug "`curl -w %{url_effective} -L -o $tempDir/$(jq '.name, " ", .browser_download_url' -r <<<$scriptAssetSHA256SUM) 2>&1 || error "Checksum download failed"`"
+  debug "`wget $releaseApiUrl -O $tempDir/$(jq '.[].name, " ", .browser_download_url' -r <<<$scriptAsset) 2>&1 || error "Release download failed"`"
```

Comment:
got error 
```
jq: error (at <stdin>:34): Cannot index string with string "name"
```
within vagrant environment

Reply:
Still looking at this one. It was working when I originally created the PR :thinking:

Reply:
This should be corrected now; needed `jq -re  '.name'...` and NOT `jq -re '.[].name'...` as it was too expanded to find the expected value